.ci/releasePipeline.groovy

```diff
@@ -26,11 +32,23 @@ pipeline {
                         script {
                             getSignClient()
 
+                            // version is without the v* prefix
                             version = getVersionFromGitTagName()
                             releaseId = createGitHubRelease(version: version)
 
+                            isFullRelease = fullRegex.matcher(version).matches()
+                            isRcRelease = rcRegex.matcher(version).matches()
+                            isDevRelease = devRegex.matcher(version).matches()
+
                             echo "version= ${version}"
+                            echo "isFullRelease=${isFullRelease}"
+                            echo "isRcRelease=${isRcRelease}"
+                            echo "isDevRelease=${isDevRelease}"
                             echo "GitHub releaseId= ${releaseId}"
+
+                            if (!isFullRelease && !isRcRelease && !isDevRelease) {
+                                error('Given version tag is not a valid tag. Use v1.2.3, v1.2.3-rc*, or v1.2.3-dev*')
```

Comment:
did you try this case out with a test tag?

Reply:
Yes, and it aborts as expected